management_api_app/api/routes/workspace_templates.py

```diff
@@ -0,0 +1,15 @@
+from fastapi import APIRouter, Depends
+
+from api.dependencies.database import get_repository
+from db.repositories.workspace_templates import WorkspaceTemplateRepository
+from models.schemas.workspace_template import WorkspaceTemplateNamesInList
+from resources import strings
+
+
+router = APIRouter()
+
+
+@router.get("/workspace-templates", response_model=WorkspaceTemplateNamesInList, name=strings.API_GET_WORKSPACE_TEMPLATES)
+async def retrieve_workspace_templates(workspace_template_repo: WorkspaceTemplateRepository = Depends(get_repository(WorkspaceTemplateRepository))) -> WorkspaceTemplateNamesInList:
```

Comment:
nit: why not just get instead of retrieve. Easier to read and means the same thing.